app/views/buyers/applications/index.html.slim

```diff
@@ -1,21 +1,16 @@
-- if @account
-  = render '/buyers/accounts/menu'
+= render '/buyers/accounts/menu'
 
-h1
-  ' Applications
-  - if @plan
-    ' on plan
-    = @plan.name
-  - elsif @service
-    ' on service
-    = @service.name
-  - elsif @account
-    ' for
-    = @account.name
+h2 = "Applications for #{@account.name}"
```

Comment:
Everything has been separated into the 3 contexts. Now instead of checking if @plan, if @service etc it renders either `provider/admin/applications/index`, `buyers/applications/index`, etc. Same for `/new`.